roles/haproxy/templates/etc/haproxy/haproxy_swift.cfg

```diff
@@ -13,14 +13,14 @@ defaults
   option httplog
   option dontlognull
   option redispatch
-  timeout client 300s
-  timeout server 300s
-  timeout connect 1s
-  timeout http-keep-alive 60s
-  timeout http-request 5s
+  timeout client {{ haproxy.defaults.timeout_client }}
+  timeout server {{ haproxy.defaults.timeout_server }}
+  timeout connect {{ haproxy.defaults.timeout_connect }}
+  timeout http-keep-alive {{ haproxy.defaults.timeout_http_keep_alive }}
+  timeout http-request {{ haproxy.defaults.timeoute_http_request }}
```

Comment:
misspelling, please change all 3 occurrences in the PR to "timeout_http_request"